.travis/after_success.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -ex
+
+if [[ $TRAVIS_OS_NAME == 'osx' ]]
+then
+    make bundle
+elif [[ $TRAVIS_OS_NAME == 'linux' ]]
```

Comment:
Can this be pushed into the `yml`? `on.condition: "$TRAVIS_OS_NAME == linux"` or does that not work

Reply:
This was the pattern already in most of the travis scripts, I don't really feel like changing it. TBH I don't understand travis's yml.